website/web/genericapi.py

```diff
@@ -389,7 +389,9 @@ class CaptureReport(Resource):  # type: ignore[misc]
     @api.param('comment', 'Description of the URL, will be given to the analyst.')  # type: ignore[misc]
     def post(self, capture_uuid: str) -> bool | dict[str, Any]:
         parameters: dict[str, Any] = request.get_json(force=True)
-        return lookyloo.send_mail(capture_uuid, parameters.get('email', ''), parameters.get('comment'))
+        from . import modules
```

Comment:
This is going to be very weird to debug and probably cause issues. Unless you have a very good reason (there are very few), you should never have to call `import` unless you're at the beginning of the file.